Update README.md (#14)
vtex-apps/store-link: docs/README.md

1. Add `store-link` app to your theme's dependencies in the `manifest.json`, for example:

```diff
  "dependencies": {
+   "vtex.store-link": "0.x"

> }

Reviewer: ```suggestion
  }
```

> }

Reviewer 2: 👀 
```suggestion
 }
```

Due to the context used by the `link.product` block, you can also build an URL path with product variables when using the `href` prop. 

| Product variable   | Description                                   |
| -------------- | --------------------------------------------- |
| `brand`      | Product brand name.                    |
| `brandId`    | Product brand ID.                     |
| `category1`  | Highest level category in the category tree.    |
| `category2`  | Second highest level category.                 |
| `category3`  | Third hieghest level category.                |
| `category4`  | Fourth highest level category.                 |
| `department` | Product department.                            |
| `productId`  | Product ID.                                    |
| `skuId`      | Current selected SKU ID.                       |
| `slug`       | The link text used to create the product link. |

Using one of these variables, you will be able to structure any desired URL for your store, such as a link to a given product department (`/{department}`).

> To build URLs with variables related to the product specifications, you should use the following format: `{specificationGroups.|{{groupName}}|.specifications.|{{specificationName}}|}`, replacing the curly brackets according to your scenario. For example:

Reviewer 2: It's a bit confusing. Maybe this?
```suggestion
To build URLs with variables related to the product specifications, you should use the following format: `{specificationGroups.GROUP NAME HERE.specifications.SPECIFICATION NAME HERE}`, replacing the curly brackets according to your scenario. For example:
```

Reviewer: Maybe `[GROUP_NAME_HERE]`?

  }
}
```

> In the example above, `App Data` is the specification group name and `vtex-url` is the product specification name.

Reviewer 2: I would change the above example to a more common one:

`{specificationGroups.Design.specifications.Dimensions}`
